src/main/webapp/resources/js/components/project/ProjectSelect.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Select, SelectProps, Tag, Typography } from "antd";
+
+export type Project = { identifier: number; name: string };
+
+export interface ProjectSelectProps extends SelectProps {
+  projects: Project[];
+}
+
+/**
+ * React component for displaying a project drop-down menu.
+ * @param {list} projects - list of projects that is to be displayed
+ * @param {function} onChange - function that is called when select option has changed
+ * @param {number} defaultValue - project identifier of the project that is to be displayed by default
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function ProjectSelect({
```

Comment:
I wonder we could generalize this to be a select for anything that has an id?

Reply:
I think so... I just need a good name. Please see d19ce2bb0c63c881f4c20b2b49984be221e6eddd.